### PR TITLE
rebuild for openssl 1.1.1,3

### PR DESCRIPTION
**Pull request by boldorider4**

### CODE REVIEW

File: recipe/build.sh [hunk 1]
```diff
@@ -8,4 +8,4 @@ if [[ $(uname) == Linux ]]; then
   export LDFLAGS="${LDFLAGS} -Wl,--no-as-needed"
 fi
 
-UWSGI_EMBED_PLUGINS=stats_pusher_statsd ${PYTHON} -m pip install . --no-deps -vv
+UWSGI_EMBED_PLUGINS=stats_pusher_statsd ${PYTHON} -m pip install . --no-deps --no-build-isolation -vvv
```
Comment (JeanChristopheMorinPerso) [inline recipe/build.sh]:
```suggestion
UWSGI_EMBED_PLUGINS=stats_pusher_statsd ${PYTHON} -m pip install . --no-deps --no-build-isolation -vv
```
`-vvv` is not helpful and just adds noise. Technically we just need a single `v` to get the output of the build.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -62,3 +62,5 @@ extra:
   recipe-maintainers:
     - kalefranz
     - ocefpaf
+  skip-lints:
+    - host_section_needs_exact_pinnings
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]: I'd remove this and would add `{{ python }}` to `libpython-static`. You'll still get the linter warning, but that's a bug and we simply need to fix it. It's actually a bug in `percy` which we already fixed. The linter just needs to be updated and it also needs to be updated on prefect.

File: recipe/build.sh (hunk 1 above)
Comment (boldorider4) [reply recipe/build.sh]: I thought I saw at least `-vv` in the majority of cases though. I will refrain from using `-vvv` then. Generally I like verbosity though.

File: recipe/meta.yaml (hunk 2 above)
Comment (boldorider4) [reply recipe/meta.yaml]: got it.

**Review by rafaelmartins — APPROVED** [review]
LGTM

**Review by Jrice1317 — APPROVED** [review]
LGTM

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**